docs/sequenceDiagrams/EditSequenceDiagramCode.txt

```diff
@@ -0,0 +1,26 @@
+title Edit
+
+
```

Comment:
Might want to include the actor here

Reply:
Oh hmmm I thought it may be better to include the actor for the parser?